src/test/kotlin/no/brreg/informasjonsforvaltning/abackendservice/utils/TestUtils.kt

```diff
@@ -1,98 +1,77 @@
 package no.brreg.informasjonsforvaltning.abackendservice.utils
 
 import java.io.BufferedReader
-import java.io.File
 import java.net.HttpURLConnection
 import java.net.URL
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
 import com.fasterxml.jackson.module.kotlin.readValue
 import org.springframework.http.HttpStatus
+import java.io.OutputStreamWriter
 
+fun apiGet(endpoint: String): Map<String,Any> {
 
-fun createTmpComposeFile(): File {
-    val tmpFile = File.createTempFile("test-compose", ".yml")
-    tmpFile.writeText("version: \"3.2\"\n" +
-        "\n" +
-        "services:\n" +
-        "  a-backend-service:\n" +
-        "    image: brreg/a-backend-service:latest\n" +
-        "    environment:\n" +
-        "      - MONGO_USERNAME=$MONGO_USER\n" +
-        "      - MONGO_PASSWORD=$MONGO_PASSWORD\n" +
-        "      - MONGO_HOST=mongodb:27017\n" +
-        "    depends_on:\n" +
-        "      - $MONGO_SERVICE_NAME\n" +
-        "\n" +
-        "  $MONGO_SERVICE_NAME:\n" +
-        "    image: mongo:latest\n" +
-        "    environment:\n" +
-        "      - MONGO_INITDB_ROOT_USERNAME=$MONGO_USER\n" +
-        "      - MONGO_INITDB_ROOT_PASSWORD=$MONGO_PASSWORD\n" )
-
-    return tmpFile
-}
-
-fun simpleGet(host: String, port: Int, address: String): String =
-    URL("http", host, port, address)
-        .openConnection()
-        .inputStream
-        .bufferedReader()
-        .use(BufferedReader::readText)
-
-fun getContent(host: String, port: Int, address: String): Map<String,Any> {
-    try {
-        val connection = URL("http", host, port, address)
+    return try{
+        val connection = URL(getApiAddress(endpoint))
                 .openConnection() as HttpURLConnection
-        connection.requestMethod = "GET"
+
         connection.connect()
 
-        return if (HttpStatus.resolve(connection.responseCode)?.is2xxSuccessful == true) {
```

Comment:
du fjernet sjekken for ok respons?

Reply:
Nei, den ligger is isOK()

Reply:
Ja, men det hjelper lite å flytte kode ut i en metode hvis du ikke bruker metode ;)